### PR TITLE
Fs 1083 fix status display

### PR DESCRIPTION
**Pull request by Adam-W1**

### CODE REVIEW

**Review by SentientHousePlant — APPROVED** [review]
LGTM 🚀

**Review by rayhan15243 — APPROVED** [review]
Have tested this in Dev. Passed QA.

**Review by harryyo — COMMENTED** [review]
Just a query about the section status switching

File: database/store.py [hunk 1]
```diff
@@ -290,6 +290,9 @@ def _update_section_statuses(self, application_id: str):
                 if application.get("date_submitted"):
                     section["status"] = "SUBMITTED"
                     break
+                elif question["status"] == "COMPLETED":
+                    section["status"] = "COMPLETED"
+                    continue
```
[the remark this answers is not in the document]
Comment (Adam-W1) [reply database/store.py:295]: the idea is that if the question is complete the loop continues to make sure that no other questions are in progress, so the section will never be set to complete unless all Q's are completed. If that makes sense? - happy to call

**Pull request merged**